### PR TITLE
Add clang-tidy warning readability-inconsistent-declaration-parameter…

### PR DESCRIPTION
**Pull request by akuzm**
…-name

Mostly cosmetic stuff. Matched to definition automatically with --fix-notes.

### CODE REVIEW

File: src/subspace_store.h [hunk 1]
```diff
@@ -23,14 +23,14 @@ extern SubspaceStore *ts_subspace_store_init(const Hyperspace *space, MemoryCont
 											 int16 max_items);
 
 /* Store an object associate with the subspace represented by a hypercube */
-extern void ts_subspace_store_add(SubspaceStore *cache, const Hypercube *hc, void *object,
+extern void ts_subspace_store_add(SubspaceStore *store, const Hypercube *hc, void *object,
```
Comment (jnidzwetzki) [inline src/subspace_store.h:26]:
```suggestion
extern void ts_subspace_store_add(SubspaceStore *subspace_store, const Hypercube *hypercube, void *object,
```

File: src/ts_catalog/tablespace.h [hunk 2]
```diff
@@ -24,7 +24,7 @@ typedef struct Tablespaces
 	Tablespace *tablespaces;
 } Tablespaces;
 
-extern Tablespace *ts_tablespaces_add(Tablespaces *tablespaces, const FormData_tablespace *form,
+extern Tablespace *ts_tablespaces_add(Tablespaces *tspcs, const FormData_tablespace *form,
```
Comment (jnidzwetzki) [inline src/ts_catalog/tablespace.h:27]:
```suggestion
extern Tablespace *ts_tablespaces_add(Tablespaces *tablespaces, const FormData_tablespace *form,
```

File: tsl/src/compression/datum_serialize.h [hunk 3]
```diff
@@ -27,24 +27,24 @@ BinaryStringEncoding datum_serializer_binary_string_encoding(DatumSerializer *se
 
 /* serialize to bytes in memory. */
 Size datum_get_bytes_size(DatumSerializer *serializer, Size start_offset, Datum val);
-char *datum_to_bytes_and_advance(DatumSerializer *serializer, char *start, Size *max_size,
-								 Datum val);
+char *datum_to_bytes_and_advance(DatumSerializer *serializer, char *ptr, Size *max_size,
```
Comment (jnidzwetzki) [inline tsl/src/compression/datum_serialize.h:30]:
```suggestion
char *datum_to_bytes_and_advance(DatumSerializer *serializer, char *start, Size *max_size,
```

File: src/chunk_constraint.h [hunk 4]
```diff
@@ -36,9 +36,9 @@ typedef struct Hypercube Hypercube;
 typedef struct ChunkScanCtx ChunkScanCtx;
 
 extern TSDLLEXPORT ChunkConstraints *ts_chunk_constraints_alloc(int size_hint, MemoryContext mctx);
-extern ChunkConstraints *ts_chunk_constraint_scan_by_chunk_id(int32 chunk_id, Size count_hint,
+extern ChunkConstraints *ts_chunk_constraint_scan_by_chunk_id(int32 chunk_id, Size num_constraints_hint,
 															  MemoryContext mctx);
-extern ChunkConstraints *ts_chunk_constraints_copy(ChunkConstraints *constraints);
+extern ChunkConstraints *ts_chunk_constraints_copy(ChunkConstraints *ccs);
```
Comment (jnidzwetzki) [inline src/chunk_constraint.h:41]:
```suggestion
extern ChunkConstraints *ts_chunk_constraints_copy(ChunkConstraints *chunk_constraints);
```

File: src/nodes/chunk_dispatch.h [hunk 5]
```diff
@@ -45,7 +45,7 @@ typedef struct Point Point;
 typedef void (*on_chunk_changed_func)(ChunkInsertState *state, void *data);
 
 extern ChunkDispatch *ts_chunk_dispatch_create(Hypertable *ht, EState *estate, int eflags);
-extern void ts_chunk_dispatch_destroy(ChunkDispatch *dispatch);
+extern void ts_chunk_dispatch_destroy(ChunkDispatch *cd);
```
Comment (jnidzwetzki) [inline src/nodes/chunk_dispatch.h:48]:
```suggestion
extern void ts_chunk_dispatch_destroy(ChunkDispatch *chunk_dispatch);
```

**Review by jnidzwetzki — CHANGES_REQUESTED** [review]
Some of the new parameter names (e.g., `char *start` -> `char *ptr`) are more difficult to understand than the old ones. It might be good if we could rename these parameters first before enabling this clang-tidy option.

File: src/chunk_index.h [hunk 6]
```diff
@@ -45,7 +45,7 @@ extern int ts_chunk_index_rename(Chunk *chunk, Oid chunk_indexrelid, const char
 extern int ts_chunk_index_rename_parent(Hypertable *ht, Oid hypertable_indexrelid,
 										const char *newname);
 extern int ts_chunk_index_adjust_meta(int32 chunk_id, const char *ht_index_name,
-									  const char *old_name, const char *new_name);
+									  const char *oldname, const char *newname);
```
Comment (fabriziomello) [inline src/chunk_index.h:48]: You renamed here the parameters and in `src/dimension.h` you made the contrary

Comment (akuzm) [reply src/chunk_index.h]: Switched to underscore everywhere for consistency.

**Review by jnidzwetzki — APPROVED** [review]
One comment added, rest looks good.

**Review by fabriziomello — APPROVED** [review]

File: src/subspace_store.h [hunk 7]
```diff
@@ -23,14 +23,14 @@ extern SubspaceStore *ts_subspace_store_init(const Hyperspace *space, MemoryCont
...
+extern void ts_subspace_store_add(SubspaceStore *subspace_store, const Hypercube *hypercube,
+								  void *object, void (*object_free)(void *));
 
 /* Get the object stored for the subspace that a point is in.
  * Return the object stored or NULL if this subspace is not in the store.
  */
-extern void *ts_subspace_store_get(const SubspaceStore *cache, const Point *target);
-extern void ts_subspace_store_free(SubspaceStore *cache);
-extern MemoryContext ts_subspace_store_mcxt(const SubspaceStore *cache);
+extern void *ts_subspace_store_get(const SubspaceStore *store, const Point *target);
```
[the remark this answers is not in the document]
Comment (akuzm) [reply src/subspace_store.h:32]: OK. I was avoiding the renames because it leads to merge conflicts and useless stuff in git blame.

**Pull request merged**